src/newgrf_spritegroup.h

```diff
@@ -378,7 +378,7 @@ struct ResolverObject {
 		this->last_value = 0;
 		this->waiting_triggers = 0;
 		this->used_triggers = 0;
-		memset(this->reseed, 0, sizeof(this->reseed));
+		std::ranges::fill(this->reseed, 0);
```

Comment:
`this->reseed.fill(0);`

Reply:
The include in newgrf.cpp is needed for `std::views::take`.